test/browser_tests/cucumber/step_definitions/reusable-text-snippets.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+
+var util = require( '../../util/index.js' );
+var shouldShouldnt = util.shouldShouldnt;
+
+var { defineSupportCode } = require( 'cucumber' );
+var { expect } = require( 'chai' );
+
+
+defineSupportCode( function( { When, Then } ) {
+
+  When( 'there exists a reusable text snippet on this page',
+    function() {
+      var rts = element( by.css( '.m-reusable-text-snippet' ) );
+      return expect( rts.isPresent() ).to.eventually.equal( true );
+    }
+  );
+
+  Then( /the snippet output (should|shouldn't) include a slug-style header/,
+    function( includeHeader ) {
+      var rts;
+
+      if ( shouldShouldnt( includeHeader ) ) {
+        rts = element.all( by.css( '.m-reusable-text-snippet' ) ).first();
```

Comment:
I understand the logic of getting .first() and .last() based on the way the test data is set up, but relying on the ordering seems a little brittle & makes the test a little harder to follow.  Would it be possible to instead check by the text inside the div? If that's too difficult feel free to ignore.